projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html

```diff
@@ -35,124 +35,159 @@ <h1 class="mb-5">
-                    'enumerationAndChronology': prediction.issue,
-                    'expected_date': prediction.expected_date
-                }">
-          <i class="fa fa-pencil"></i>
-        </button>
-      </div>
-    </div>
-    <!-- RECEIVED ISSUES ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
-    <div class="row mt-1 "
-         *ngFor="let item of this.receivedItems"
-         [ngClass]="{'deleted-status': item.metadata.issue.status == 'deleted'}">
-      <div class="col-sm-4 text">
-        {{ item.metadata.enumerationAndChronology }}
-        <span class="badge badge-pill badge-info pl-2" *ngIf="item.new_issue" translate>New</span>
-      </div>
-      <div class="col-sm-1 text-center">
-        <i class="fa" title="{{ item.metadata.issue.status | translate }}"
-           [ngClass]="getIcon(item.metadata.issue.status)"
-        ></i>
-      </div>
-      <div class="col-sm-2 text">{{ item.metadata.call_number }}</div>
-      <div class="col-sm-3 text">{{ item.metadata.issue.received_date | dateTranslate }}</div>
-      <div class="col-sm-2 text-right">
-        <!-- DETAIL BUTTON -->
-        <button type="button"  class="btn btn-sm btn-outline-primary"
-                title="{{ 'Detail' | translate }}"
-                [routerLink]="['/records', 'items', 'detail', item.metadata.pid]">
-          <i class="fa fa-eye"></i>
-        </button>
-        <!-- EDIT BUTTON -->
-        <button *ngIf="item.permissions && item.permissions.update && item.permissions.update.can"
-                type="button" class="btn btn-sm btn-outline-primary ml-1"
-                title="{{ 'Edit' | translate}}"
-                [routerLink]="['/records', 'items', 'edit', item.metadata.pid]"
-                [queryParams]="{'redirectTo': 'records/holdings/detail/' + holding.id }">
-          <i class="fa fa-pencil"></i>
-        </button>
-        <!-- DELETE BUTTON -->
-        <button *ngIf="item.permissions && item.permissions.delete && item.permissions.delete.can; else deleteInfo"
-                type="button" class="btn btn-outline-danger btn-sm ml-1"
-                title="{{ 'Delete' | translate}}"
-                (click)="deleteIssue(item)">
-          <i class="fa fa-trash" ></i>
-        </button>
-        <ng-template #deleteInfo>
-          <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
-                  title="{{ 'Delete' | translate}}"
-                  [popover]="tolTemplate" triggers="mouseenter:mouseleave">
-            <i class="fa fa-trash"></i>
+<!-- TABS -->
+<section class="mt-3 mb-4">
+  <tabset>
+    <!-- ISSUES TAB ------------------------------------------------------------>
+    <tab id="holding-local-field-tab" tabOrder="1">
+      <ng-template tabHeading>
+        <i class="fa fa-list-ul mr-1"></i><span translate>Issues</span>
+      </ng-template>
+      <div class="row mt-2">
+        <div class="col-12 mt-1">
```

Comment:
col-12 -> col, you can search for similar cases.

Reply:
done